include/json/config.h

```diff
@@ -67,21 +67,40 @@
 // Storages, and 64 bits integer support is disabled.
 // #define JSON_NO_INT64 1
 
-#if defined(_MSC_VER) && _MSC_VER <= 1200 // MSVC 6
-// Microsoft Visual Studio 6 only support conversion from __int64 to double
-// (no conversion from unsigned __int64).
-#define JSON_USE_INT64_DOUBLE_CONVERSION 1
-// Disable warning 4786 for VS6 caused by STL (identifier was truncated to '255'
-// characters in the debug information)
-// All projects I've ever seen with VS6 were using this globally (not bothering
-// with pragma push/pop).
-#pragma warning(disable : 4786)
-#endif // if defined(_MSC_VER)  &&  _MSC_VER < 1200 // MSVC 6
-
-#if defined(_MSC_VER) && _MSC_VER >= 1500 // MSVC 2008
-/// Indicates that the following function is deprecated.
-#define JSONCPP_DEPRECATED(message) __declspec(deprecated(message))
-#endif
+#if defined(_MSC_VER) // MSVC
+	#if _MSC_VER <= 1200 // MSVC 6
```

Comment:
I don't think the indents as written improve the clarity of the file overall.

The usual thing is to keep the `#` in column 1. This is historical convention.
The next token is indented by 0, 1, or 2 spaces. In this project, it's 0, but you've chosen 8.

The idea is to keep the preprocessor lines visually segregated from the C++ code.
A reader can eyeball column 1 and see a run of `#`, which is easy.